### PR TITLE
Add component for claims

### PR DESCRIPTION
**Pull request by xrchz**
Get claimable amounts for each interval, and node minipools

### CODE REVIEW

File: react-ui/src/components/ClaimIntervals.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    useContractReads({
+        enabled: nodeAddress && rocketMinipoolManagerAddress && validatingMinipoolCount,
+        contracts: Array.from(Array(validatingMinipoolCount).keys()).map(i =>
+          ({address: rocketMinipoolManagerAddress,
+            abi: minipoolManagerAbi,
+            functionName: "getNodeValidatingMinipoolAt",
+            args: [nodeAddress, i]})),
+        onSuccess: (data) => setNodeMinipools(data)
+    })
+
```
Comment (nickgs) [inline react-ui/src/components/ClaimIntervals.js:105]: This looks great. What we can do to test initially is add a small return value here that displays some of the values collected from the contract calls.

File: react-ui/src/components/ClaimIntervals.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        onSuccess: (result) => {
+            setCurrentIntervalIndex(result);
+        }
+    })
+
+    useContractRead({
+        address: rocketMinipoolManagerAddress,
+        enabled: nodeAddress && rocketMinipoolManagerAddress,
+        abi: minipoolManagerAbi,
+        functionName: "getNodeValidatingMinipoolCount",
```
Comment (nickgs) [inline react-ui/src/components/ClaimIntervals.js]: I think you are correct, we need to change this to use getNodeMinipoolCount and getNodeMinipoolAt. This way we collect all minipools associated with this node that may have balances.

**Review by nickgs — COMMENTED** [review]
This looks good. I made a couple of small comments. The overall direction is solid.

File: react-ui/src/components/ClaimIntervals.js (hunk 1 above)
Comment (xrchz) [reply react-ui/src/components/ClaimIntervals.js:105]: Done

**Pull request merged**